controllers/base_controller.js

```diff
@@ -117,7 +117,7 @@ module.exports = function BaseControllerModule(pb) {
         this.body                = props.body;
         this.localizationService = props.localization_service;
         this.ls                  = this.localizationService;
-        this.pathVars            = props.path_vars;
+        this.pathVars            = props.pathVars;
```

Comment:
trying standardize on no underscores.  Duplicated the property until to deprecate the other property